l10n_it_corrispettivi/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 {
     'name': 'Italian Localization - Corrispettivi',
-    'version': '10.0.1.2.2',
+    'version': '10.0.1.3.0',
```

Comment:
come mai non `10.0.1.2.3`?
Mi pare ricada in:
`z increments when bugfixes were made. Usually a server restart is needed for the fixes to be made available.`(cit. [OCA version-numbers](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers))

Reply:
Penso anch'io

Reply:
@tafaRU grazie della review! In effetti ero indeciso, ho messo `10.0.1.2.3`